migration/v6/model/pkg.go

```diff
@@ -0,0 +1,79 @@
+package model
+
+import "github.com/konveyor/tackle2-hub/migration/v5/model"
+
+//
+// JSON field (data) type.
+type JSON = []byte
+
+type Model = model.Model
+type Application = model.Application
+type Bucket = model.Bucket
+type BucketOwner = model.BucketOwner
+type BusinessService = model.BusinessService
+type Dependency = model.Dependency
+type File = model.File
+type Fact = model.Fact
+type Identity = model.Identity
+type Import = model.Import
+type ImportSummary = model.ImportSummary
+type ImportTag = model.ImportTag
+type JobFunction = model.JobFunction
+type MigrationWave = model.MigrationWave
+type Proxy = model.Proxy
+type Review = model.Review
+type Setting = model.Setting
+type RuleSet = model.RuleSet
+type Rule = model.Rule
+type Stakeholder = model.Stakeholder
+type StakeholderGroup = model.StakeholderGroup
+type Tag = model.Tag
+type TagCategory = model.TagCategory
+type Task = model.Task
+type TaskGroup = model.TaskGroup
+type TaskReport = model.TaskReport
+type Ticket = model.Ticket
+type Tracker = model.Tracker
+type TTL = model.TTL
+type ApplicationTag = model.ApplicationTag
+type DependencyCyclicError = model.DependencyCyclicError
+
+//
+// All builds all models.
+// Models are enumerated such that each are listed after
+// all the other models on which they may depend.
+func All() []interface{} {
```

Comment:
Looks like the tool puts these in a semi-alphabetical order rather than dependency order. This usually won't cause a problem unless we're adding new models or foreign key relationships, but it makes the comment inaccurate.

Reply:
replace with All() from v5.